src/components/markdown-render/inlineCode.js

```diff
@@ -1,6 +1,6 @@
 import React from 'react';
 import Highlight, { defaultProps } from 'prism-react-renderer';
-import customTheme from 'prism-react-renderer/themes/github';
+import customTheme from 'prism-react-renderer/themes/duotoneDark';
```

Comment:
This change mentions that this creates a *higher* contrast, but was it a necessary change?

Did the github theme not have adequate enough contrast? If it doesn't, then I'm all in for some kind of change.

We chose this theme because it matched our branding the best. Is there a different theme that has good contrast and matches our theme?

Reply:
We ended up switching to a color scheme based on the Spark colors.